aws_lambda_fsm/fsm.py

```diff
@@ -755,7 +757,7 @@ def dispatch(self, event, obj):
                 # understand fence tokens.
 
                 # make the fence token available
-                if isinstance(fence_token, (int, long)):
```

Comment:
this is also broken for `0L`

Reply:
`int` here is from Futurize, and it mimics `int` in Python 3. See [here](http://python-future.org/compatible_idioms.html#long-integers).